<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:#552a59;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tableWidget">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>781</width>
      <height>511</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Poppins Medium</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;
</string>
    </property>
    <column>
     <property name="text">
      <string>First Name</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Last Name</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>ID</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Role</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Total Points</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Session Points</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Absences Strike</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>30</y>
      <width>161</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Poppins SemiBold</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;
</string>
    </property>
    <property name="text">
     <string>Students List</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(Data/InterfaceData/back.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="addButton">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>20</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:url(Data/InterfaceData/add.png);
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="resetSession">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>40</y>
      <width>161</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Poppins</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:10;
background-color: rgb(252, 67, 67);</string>
    </property>
    <property name="text">
     <string>Reset Session Points</string>
    </property>
   </widget>
   <widget class="QPushButton" name="resetAbs">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>10</y>
      <width>161</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Poppins</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:10;
background-color: rgb(252, 67, 67);</string>
    </property>
    <property name="text">
     <string>Reset Absences</string>
    </property>
   </widget>
   <widget class="QPushButton" name="resetPre">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>25</y>
      <width>161</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Poppins</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:10;
background-color: rgb(12, 244, 244);</string>
    </property>
    <property name="text">
     <string>Reset Presents List</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
